src/main/java/com/nukkitx/proxypass/network/ProxyBedrockEventHandler.java

```diff
@@ -31,6 +30,11 @@
         ADVERTISEMENT.setSubMotd("https://github.com/NukkitX/ProxyPass");
     }
 
+    public ProxyBedrockEventHandler(ProxyPass proxy) {
+        this.proxy = proxy;
+        ADVERTISEMENT.setIpv4Port(this.proxy.getProxyAddress().getPort());
```

Comment:
We bind to IPv6 on the same port so it can be set to the same.

Reply:
Fixed in a3ee944.